### PR TITLE
Create a super build for LiAgent

### PR DESCRIPTION
**Pull request by amarpad**
## Summary

Change creates a super build for LiAgent
Also, fixes the build script to capture the new updated path for connection tracker

## Test Plan

Build all, verified sym links are correct

Signed-off-by: Amar Padmanabhan <amar@freedomfi.com>

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

**Comment by amarpad** [comment]
@ymasmoudi looks like build-magma.sh is not copying liagentd?

File: lte/gateway/c/li_agent/src/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+# we can find memory leaks in production.
+set(CMAKE_C_FLAGS_RELWITHDEBINFO
+    "${CMAKE_C_FLAGS_RELWITHDEBINFO} -fsanitize=leak -fno-omit-frame-pointer")
+set(CMAKE_EXE_LINKER_FLAGS_RELWITHDEBINFO
+    "${CMAKE_EXE_LINKER_FLAGS_RELWITHDEBINFO} -fsanitize=leak")
+
+add_definitions(-DLOG_WITH_GLOG)
+
+# compile the needed macros
+create_proto_dir("lte" LTE_CPP_OUT_DIR)
```
Comment (themarwhal) [inline lte/gateway/c/li_agent/src/CMakeLists.txt:47]: this is unrelated to your change but i think we don't need `spgw_service` for this service? or do we? or policydb :o

**Comment by themarwhal** [comment]
Does this fix https://github.com/magma/magma/issues/7518 by any chance? (maybe it is unrelated as it happens during session manager build)

File: lte/gateway/c/li_agent/src/CMakeLists.txt (hunk 1 above)
Comment (amarpad) [reply lte/gateway/c/li_agent/src/CMakeLists.txt:47]: Let me take a look. It seemed like session_manager added a dependency into policydb.

**Comment by amarpad** [comment]
> Does this fix #7518 by any chance? (maybe it is unrelated as it happens during session manager build)

I suspect not.

File: lte/gateway/c/li_agent/src/CMakeLists.txt (hunk 1 above)
Comment (amarpad) [reply lte/gateway/c/li_agent/src/CMakeLists.txt:47]: ok so mobilityd client brings in subscriberdb which brings in APN. Thats the minimum set of protos we need. Fixed and updated, thanks for the review.

Comment (themarwhal) [reply lte/gateway/c/li_agent/src/CMakeLists.txt:47]: thanks! that was bothering me for a bit :)

**Review by koolzz — APPROVED** [review]
cc @ymasmoudi

**Review by ardzoht — APPROVED** [review]

**Comment by themarwhal** [comment]
@amarpad locally I don't see this path "${C_BUILD}/connection_tracker/src/src" but I see "${C_BUILD}/connection_tracker/src"

I think this is breaking the LTE deploy job: https://github.com/magma/magma/issues/7601

**Pull request merged**